index.html

```diff
@@ -0,0 +1,303 @@
+                    or sign-ups required.
+                  </p>
+                </div>
+                <div class="language-list-holder">
+                  <ul class="language-container">
+                    <li class="language-item"><span>Html</span></li>
+                    <li class="language-item"><span>Css</span></li>
+                    <li class="language-item"><span>JavaScript</span></li>
+                  </ul>
+                </div>
+                <button type="button" class="butto">See project</button>
+              </section>
+            </li>
+          </ul>
+        </div>
+  
+        <div class="box-contain">
+          <ul>
+            <li class="ptflio-item">
+              <img
+                class="ptflio-image"
+                src="./images/Snapshoot-Portfolio4.png"
+                alt="Portfolio"
+              />
+              <section>
+                <div class="lists-container">
+                  <h2>Multi-Post <br> Stories</h2>
+                  <div class="lists">
+                    <span class="lists-item obect1">CANOPY</span>
+                    <img src="./images/Counter.png" alt="Dot Counter" />
+                    <span class="lists-item">back end Dev</span>
+                    <img src="./images/Counter.png" alt="Dot Counter" />
+                    <span class="lists-item">2015</span>
+                  </div>
+                </div>
+                <div class="text-holder">
+                  <p>
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
+                  </p>
+                </div>
+                <div class="language-list-holder">
+                  <ul class="language-container">
+                    <li class="language-item"><span>Html</span></li>
+                    <li class="language-item"><span>Css</span></li>
+                    <li class="language-item"><span>JavaScript</span></li>
+                  </ul>
+                </div>
+                <button type="button" class="butto">See project</button>
+              </section>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-holder">
+          <section>
+            <h2 class="head">About Myself</h2>
+            <p class="helloo">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h4 class="ico">LET’S CONNECT</h4>
+            <ul class="ico-img">
+              <li>
+                <a href="#"
+                  ><img
+                    class="t-social-media"
+                    src="./images/github.png"
+                    alt="twitter"
+                /></a>
+              </li>
```

Comment:
- [ ] Dear @Hirwacardinalloic, I'm impressed with what you have just done so right now. One thing left to be addressed to be exact with the Figma design is that your design overflows on a `375px` screen size. Please make sure to add the margins (top, bottom, left, right) defined in the design. 

### Figma design 
![background2](https://user-images.githubusercontent.com/69077061/205461140-f6086be9-367d-4ab2-8c92-86de375e6538.PNG)

### Your design on `375px` screen
![background](https://user-images.githubusercontent.com/69077061/205461150-fa35f2ff-9689-489a-b912-d812eee906d5.PNG)